projects/composition/src/app/pages/autocomplete-page/autocomplete-page.component.html

```diff
@@ -142,6 +142,21 @@
         prefixIcon="search"
         appearance="borderless">
       </cps-autocomplete>
+
+      <cps-autocomplete
+        label="Autocomplete with async validation"
+        [options]="options"
+        optionLabel="name"
+        optionInfo="info"
+        placeholder="Select a city"
+        [clearable]="true"
+        [(ngModel)]="selectedOption"
+        [ngModelOptions]="{ standalone: true }"
+        [validating]="validating"
```

Comment:
### Bug Report: Aquduct Bug

**Description:**
To trigger the Aquduct bug, change the `validating` property to `loading`.

**Steps to Reproduce:**
1. Change the `validating` property to `loading`.
2. After loading it causes the menu to pop up.

**Example:**

```html
<cps-autocomplete
    label="Autocomplete with async validation"
    [options]="options"
    optionLabel="name"
    optionInfo="info"
    placeholder="Select a city"
    [clearable]="true"
    [(ngModel)]="selectedOption"
    [ngModelOptions]="{ standalone: true }"
    [loading]="validating"
    (valueChanged)="onOptionSelected($event)"
    [externalError]="externalError"
    hint="This autocomplete simulates async validation upon selection.">
</cps-autocomplete>
```

the logic that tiriggers this is found in `cps-autocomplete.component.ts` here:
```typescript
ngOnChanges(changes: SimpleChanges): void {
    if (changes.options) {
      this.filteredOptions = this.options;
      this.recalcVirtualListHeight();
    }
    const loadChanges = changes.loading;
    if (
      loadChanges &&
      !loadChanges.firstChange &&
      !loadChanges.currentValue &&
      loadChanges.previousValue
    ) {
      this._toggleOptions(true);
    }
  }
```

Reply:
Oh, I see. We should use validating instead of loading in our Aqueduct component. Loading is for a different functionality.  :)